src/ms-review.tex

```diff
@@ -0,0 +1,1786 @@
+multiple datasets and thus provide links, allowing e.g., to do a joint fit to
+multiple IACT datasets, or to a joint IACT and \fermi dataset. Many
+examples are given in the tutorials.
+
+\todo{Add fit code example...?}
+
+The Fit class provides a uniform interface to multiple fitting backends:
+“minuit”~\citep{iminuit}, “scipy”,~\citep{2020SciPy-NMeth}, and
+“sherpa”~\citep{sherpa-2005,sherpa-2011}. Note that, for now, covariance matrix
+and errors are computed only for the fitting with MINUIT. However depending on
+the problem other optimizers can better perform, so sometimes it can be useful
+to run a pre-fit with alternative optimization methods. In future we plan to
+extend the supported fitting backends, including for example MCMC solutions.
+\footnote{a prototype is available in gammapy-recipes,
+	\url{https://gammapy.github.io/gammapy-recipes/_build/html/notebooks/mcmc-sampling-emcee/mcmc_sampling.html}
+}
+
+\subsection{gammapy.estimators}
+\label{ssec:gammapy-estimators}
+By fitting parametric models to the data, the total integrated
+flux and overall temporal, spectral and morphological shape of the
+\gammaray emission can be constrained. In many cases it useful
+to make a more detailed follow-up analysis by measuring the
+flux in smaller spectral, temporal or spatial bins. This
+possibly reveals more detailed emission features, which
+are relevant for studying correlation with counterpart emissions.
+
+The \code{gammapy.estimators} sub-module features methods to compute flux
+points, light curves, flux maps, flux profiles from data.
+The basic method for all these measurements is equivalent.
+The initial fine bins of \code{MapDataset} are grouped into
+larger bins. The norm of the best fit "reference" spectral
+model is fitted ("scaled") in the  restricted data range only.
+
+The base class of all algorithms is the \code{Estimator}  class.
+
+Signifiance is estimated based on hypothesis
+testing against the background only model. This allows
+to assign a significance to the given flux bin.
+In addition to the best fit flux norm all estimators compute
+quantities corresponding to this flux. This includes
+the predicted number of total, signal and background
+counts per flux bin. The total fit statistics
+of the best fit model, the fit statistics of the
+null hypothesis and the difference between both,
+the so-called TS value.
+From the TS value the significance of the
+significance of the measured signal and associated flux
+can be derived.
+
+Optionally it can also compute more advanced quantities
+such as assymmetric flux errors, flux upper limits
+and one dimensional profiles of the fit statistic,
+which show how the likelihood functions varies with
+the flux norm parameter around the fit minimum.
+This information is useful for inspecting the quality
+of the fit, where assumptoticcally a parabolic
+shape of the profile is expected at the best fit
+values (\todo{Reference to modeling / fitting section?}).
```

Comment:
I have even not sure that in our documentation we speak about this statistical properties... Maybe one could create an issue for the document.
The demonstration can be found in:
- Wilks 1962 ($13.8)
- Eadie et al. 1971 ($10.5.2)
- Li 1980 ($6.3.4)
I found these references in the Li&Ma paper.